src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,15 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="hibernate.dialect">org.hibernate.dialect.MySQL8Dialect</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/ticket_app?serverTimeZone=UTC</property>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.username">dev</property>
+        <property name="connection.password">Quaci_2sia0Me</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.hibernate.relations.model.Country" />
+        <mapping class="mate.academy.hibernate.relations.model.Actor" />
+        <mapping class="mate.academy.hibernate.relations.model.Movie" />
+    </session-factory>
+</hibernate-configuration>
```

Comment:
no empty line at the end of file

Reply:
Thank's for review
I will try to write more meanfull comments in the future :)